source/Halibut/Transport/Proxy/HttpProxyClient.cs

```diff
@@ -231,6 +232,7 @@ public TcpClient CreateConnection(string destinationHost, int destinationPort, T
                     // attempt to open the connection
                     log.Write(EventType.Diagnostic, "Connecting to proxy at {0}:{1}", ProxyHost, ProxyPort);
                     TcpClient.ConnectWithTimeout(ProxyHost, ProxyPort, timeout, cancellationToken);
+                    log.Write(EventType.Diagnostic, "Connected to proxy at {0}:{1}", ProxyHost, ProxyPort);
```

Comment:
Helpful when determining where the http proxy client got stuck

Reply:
Was it getting stuck?

Reply:
The connect was not, that timeout worked. It was the read/write after that got stuck. This is just helpful in the logs as we see it is trying to connect and then it did connect